packages/README.md

```diff
@@ -115,6 +117,10 @@ npm install glob --save-dev
 
 This commands adds the latest version of `glob` as a development dependency to the `package.json` file. It has to be executed from the root of the project.
 
+## Maintaining a Public API
```

Comment:
It looks like you could put this paragraph under the next section which is also about API 😄 

We can also create a connection with the next paragraph but putting emphasis that generated API helps to find out what's actually becomes Public API.